lang/fr/block_navbuttons.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+// This file is part of the Navigation buttons plugin for Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
```

Comment:
This all looks OK to me, but I can't merge this without also pulling in all the other changes in the branch (some of which are unwanted, some of which need fixing).